zipkin-ui/webpack.config.js

```diff
@@ -27,9 +27,9 @@ module.exports = {
         }]
     },
     output: {
-        path: __dirname + '/dist/static',
+        path: __dirname + '/build/resources/main/static/assets',
```

Comment:
Ok, I see. Is it possible to set HtmlWebpackPlugin's filename in terms of __dirname maybe? It doesn't feel quite right to put a generated resource in the parent directory of where it's supposed to be :-/

Before we merge this, maybe we could open a PR on zipkin-query and zipkin-java to see what the corresponding diff would be there? I'm not so convinced it will be so much easier, but then again I haven't looked to much at it, so I might be wrong. I'm a bit -1 for this change, unless it gives us real benefits on the other side...